orchagent/intfsorch.h

```diff
@@ -16,8 +16,7 @@ class IntfsOrch : public Orch
 public:
     IntfsOrch(DBConnector *db, string tableName, PortsOrch *portsOrch);
 private:
-    void doTask();
-
+    virtual void doTask(_in_ Consumer& consumer_info);
```

Comment:
because base class is virtual. putting here virtual is to make it explicit and avoid any possible confusion.